bubble_sort.rb

```diff
@@ -0,0 +1,35 @@
+def bubble_sort(arr)  
+  arr.each { |element|
+    arr.each_with_index { |element,index|
+    unless index == arr.length - 1
+    if arr[index] > arr[index + 1]        
+      tmp = arr[index]
+      arr[index] = arr[index + 1]
+      arr[index+1] = tmp
+    end
+    end
+    }
+  }
+end
+
+def bubble_sort_by (array)
+  for i in 0...array.length
+  for j in 0...array.length - 1 
+    check=yield(array[j],array[j + 1])
```

Comment:
Kindly know that one the Microverse guidelines encourage students to leave space around operators [read more](https://github.com/github/rubocop-github/blob/master/STYLEGUIDE.md#ruby-style-guide)
And you can see the linter also caught that......kindly fix it.